docs/src/main/sphinx/connector/redis.rst

```diff
@@ -140,6 +141,19 @@ files, which must end with ``.json`` and contain table description files.
 
 This property is optional; the default is ``etc/redis``.
 
+``redis.table-description-cache-ttl``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When adding a file end with ``.json`` to ``redis.table-description-dir`` folder,
+the Redis connector dynamically loads the file after waiting for the time
+specified by this parameter, without modifying the ``redis.table-names`` parameter
+and restarting the Trino service.
+
+Make sure to deploy same table description files to all nodes, if those files are
+different among nodes, it may cause inconsistent results for multiple queries.
```

Comment:
> it may cause inconsistent results for multiple queries

This implies this PR is no-go.

Reply:
I have researched and tested carefully. 
When querying Redis data through Trino, only the table description files of the coordinator node will be used, and the table description files of the worker nodes will not be used. Therefore, if those files are different among nodes, it will not affect the query results of multiple queries. 
Do you think my understanding is correct? If you think this is right, I will modify the documentation. Thank you very much!

Reply:
I think correct. Please update the documentation.

Reply:
Thank you very much, I updated the documentation, what do you think of this?